src/app/menu/menu.component.scss

```diff
@@ -0,0 +1,3 @@
+mat-grid-tile:hover {
+  background-color: #9E9E9E;

```

Comment:
please try to use hsl color to make hovered a little bit darker

Reply:
https://codesalad.dev/blog/color-manipulation-with-css-variables-and-hsl-16

Reply:
experiments with hsl - done